angr/analyses/veritesting.py

```diff
@@ -333,76 +343,7 @@ def _execute_and_merge(self, path):
-                        continue
-
-                    # let everyone know of the impending disaster
-                    l.info("Merging %d paths in stash %s", stash_size, stash_name)
-
-                    # Try to prune the stash, so unsatisfiable paths will be thrown away
-                    path_group.prune(from_stash=stash_name, to_stash='pruned')
-                    if 'pruned' in path_group.stashes and len(path_group.pruned):
-                        l.debug('... pruned %d paths from stash %s', len(path_group.pruned), stash_name)
-                    # Remove the pruned stash to save memory
-                    path_group.drop(stash='pruned')
-
-                    # merge things callstack by callstack
-                    while len(path_group.stashes[stash_name]):
-                        r = path_group.stashes[stash_name][0]
-                        path_group.move(
-                            stash_name, 'merge_tmp',
-                            lambda p: p.callstack == r.callstack #pylint:disable=cell-var-from-loop
-                        )
-
-                        old_count = len(path_group.merge_tmp)
-                        l.debug("... trying to merge %d paths.", old_count)
-
-                        # merge the loop_ctrs
-                        new_loop_ctrs = defaultdict(int)
-                        for m in path_group.merge_tmp:
-                            for head_addr, looping_times in m.info['loop_ctrs'].iteritems():
-                                new_loop_ctrs[head_addr] = max(
-                                    looping_times,
-                                    m.info['loop_ctrs'][head_addr]
-                                )
-
-                        path_group.merge(stash='merge_tmp')
-                        for m in path_group.merge_tmp:
-                            m.info['loop_ctrs'] = new_loop_ctrs
-
-                        new_count = len(path_group.stashes['merge_tmp'])
-                        l.debug("... after merge: %d paths.", new_count)
-
-                        merged_anything |= new_count != old_count
-
-                        if len(path_group.merge_tmp) > 1:
-                            l.warning("More than 1 path after Veritesting merge.")
-                            path_group.move('merge_tmp', 'active')
-                        elif any(
-                            loop_ctr >= self._loop_unrolling_limit + 1 for loop_ctr in
-                            path_group.one_merge_tmp.info['loop_ctrs'].itervalues()
-                        ):
-                            l.debug("... merged path is overlooping")
-                            path_group.move('merge_tmp', 'deadended')
-                        else:
-                            l.debug('... merged path going to active stash')
-                            path_group.move('merge_tmp', 'active')
-
+                path_goup = _join_merge_points(path_group, merge_points)
```

Comment:
`path_goup` -> `path_group`

And I think you missed a `self.` before `_join_merge_points(`.